testing/README.md

```diff
@@ -0,0 +1,75 @@
+### E2E tests
+
+#### Dependencies
+
+[oblt-cli](https://studious-disco-k66oojq.pages.github.io)
+
+### Try it locally
+
+0. Setup vault
+Export the environment variable `VAULT_TOKEN` or have the `~/.vault-token` file created.
```

Comment:
Can you add a better description of the vault token and how to generate it. The link to the vault docs eventually references the vault token but there is no header explaining what it does or why you would want it.

Also per my comment below this information probably should not be in a public repository.

Reply:
Should we remove the vault token information? It related to `oblt-cli` only, it's an internal tool

Reply:
Yes

Reply:
So will add the token information to the private repo